openbb_terminal/etf/etf_controller.py

```diff
@@ -312,27 +312,25 @@ def call_load(self, other_args: List[str]):
             if holdings.empty:
                 console.print("No company holdings found!\n")
             else:
-                self.etf_holdings = holdings.index[: ns_parser.limit].tolist()
-
-                if "N/A" in self.etf_holdings:
-                    na_tix_idx = [
-                        str(idx)
-                        for idx, item in enumerate(self.etf_holdings)
-                        if item == "N/A"
-                    ]
-
-                    console.print(
-                        f"N/A tickers found at position {','.join(na_tix_idx)}. "
-                        "Dropping these from holdings.\n"
-                    )
-
-                self.etf_holdings = list(
-                    filter(lambda x: x != "N/A", self.etf_holdings)
-                )
-
-                console.print(
-                    f"Top company holdings found: {', '.join(self.etf_holdings)}"
-                )
+                console.print("Top holdings found:")
+                for val in holdings["Name"].values[: ns_parser.limit].tolist():
+                    console.print(f"   {val}")
+
+                for tick, name in zip(
+                    holdings.index[: ns_parser.limit].tolist(),
+                    holdings["Name"].values[: ns_parser.limit].tolist(),
+                ):
+                    if tick != "N/A":
+                        if (
+                            "ETF" not in name
+                            and "Future" not in name
+                            and "Bill" not in name
+                            and "Portfolio" not in name
```

Comment:
There are a lot more edge cases:

This would not handle this one

https://stockanalysis.com/etf/aaa/holdings/

Reply:
Will handle this one. Let me know if you find any that I'm missing

Reply:
https://stockanalysis.com/etf/aaau/holdings/

Probably will be the same for any commodity

Reply:
https://stockanalysis.com/etf/acio/holdings/

has "Option"

Reply:
https://stockanalysis.com/etf/adfi/holdings/

Has "Bd" instead of bond.  Also has us dollars

HAvent even gotten past "AD" in the alphabet

Reply:
https://stockanalysis.com/etf/adme/holdings/

has some fun ones

Reply:
What do you mean? These are working as expected

<img width="918" alt="Screenshot 2022-10-31 at 14 48 52" src="https://user-images.githubusercontent.com/25267873/199036781-33f02230-4d2b-452d-aa7e-960002140227.png">